src/common/tests.rs

```diff
@@ -27,6 +27,7 @@ fn test_error_convert() {
     match e {
         CoseError::EncodeFailed => {
             assert!(format!("{:?}", e).contains("encode CBOR failure"));
+            assert!(format!("{}", e).contains("CBOR encoding failure"));
```

Comment:
This PR drops the code coverage under test from 97.87% to 96.86%.  It would be nice to keep the rate up, but adding a test that displays all the errors would be tedious and only really act as a change-detector.  

Maybe adding something like:
```rust
  // Exercise `Display` for the error type while we're here.
  let _ = format!("{}", err);
```
 to `util::expect_err()` might work?